Harvest.Web/ClientApp/src/Tickets/AllTickets.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { Project, Ticket } from "../types";
+import { ProjectHeader } from "../Shared/ProjectHeader";
+import { TicketTable } from "./TicketTable";
+
+interface RouteParams {
+  projectId?: string;
+}
+
+export const AllTickets = () => {
+  const { projectId } = useParams<RouteParams>();
+  const [project, setProject] = useState<Project>();
+  const [tickets, setTickets] = useState<Ticket[]>([]);
+
+
+  useEffect(() => {
+    const cb = async () => {
+      const response = await fetch(`/Project/Get/${projectId}`);
+
+      if (response.ok) {
+        const proj: Project = await response.json();
+        setProject(proj);
+      }
+    };
+
+    cb();
+  }, [projectId]);
+
+  useEffect(() => {
```

Comment:
you should make this rely on the projectId just like the other useEffect.  If the projectId changes, you definitely need to fetch new tickets, you can't just do it once.

Reply:
@srkirkland Do you mean just put the `, [projectId]` back in this one like it is above?

Reply:
yes